app/views/comments/_form.html.erb

```diff
@@ -1,10 +1,13 @@
-<%= form_with model: [current_project, @comment], data: { behavior: 'rich-toolbar' } do |f| %>
+<%= form_with model: [current_project, @comment] do |f| %>
   <%= f.text_area :content,
                   value: @comment.content,
                   rows: 5,
                   required: 'required',
                   class: 'form-control mt-3',
-                  data: { behavior: 'mentionable' } %>
+                  data: {
+                    behavior: 'mentionable rich-toolbar',
+                    'rich-toolbar': 'block-code bold italic link list-ol list-ul table'
```

Comment:
I think this attribute name could be more descriptive, perhaps `rt-buttons` or something? Also, what about nesting multiple?

```ruby
data: {
  behavior: 'mentionable rich-toolbar',
  rt: {
    buttons: 'block-code bold italic'
  }
}
```

Would Rails turn that into: `data-rt-buttons`?

Reply:
Posted in another thread but it won't nest as we would like it to.